examples/python/pincell_depletion/run_depletion.py

```diff
@@ -0,0 +1,161 @@
+import openmc
+import numpy as np
+
+###############################################################################
+#                      Simulation Input File Parameters
+###############################################################################
+
+# OpenMC simulation parameters
+batches = 100
+inactive = 10
+particles = 1000
+
+# Depletion simulation parameters
+time_step = 1.*24*60*60 # s
+final_time = 15.*24*60*60 # s
+time_steps = np.full(np.int(final_time / time_step), time_step)
+
+chain_file = './chain_simple.xml'
+power = 174 # W/cm, for 2D simulations only (use W for 3D)
+
+###############################################################################
+#                              Define materials
+###############################################################################
+
+# Instantiate some Materials and register the appropriate Nuclides
+uo2 = openmc.Material(material_id=1, name='UO2 fuel at 2.4% wt enrichment')
+uo2.set_density('g/cm3', 10.29769)
+uo2.add_element('U', 1., enrichment=2.4)
+uo2.add_element('O', 2.)
+uo2.depletable = True
+
+helium = openmc.Material(material_id=2, name='Helium for gap')
+helium.set_density('g/cm3', 0.001598)
+helium.add_element('He', 2.4044e-4)
+helium.depletable = False
```

Comment:
Normally only fuel materials would be depleted. Are you sure you want this?

Reply:
Not really. Deleting for sake of brevity